astroquery/mast/collections.py

```diff
@@ -141,10 +143,17 @@ def query_region_async(self, coordinates, radius=0.2*u.deg, catalog="Hsc",
                 if version == 1:
                     service = "Mast.Catalogs.GaiaDR1.Cone"
                 else:
-                    if version not in (2, None):
+                    if version not in (None, 2):
```

Comment:
why this change?

Reply:
it made more sense to me to have `None` go first